GooglePlayInstant/Samples/TestApp/Scripts/TestApp.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+
+namespace GooglePlayInstant.Samples.TestApp
+{
+    /// <summary>
+    /// Tests instant app plugin features through button clicks
+    /// </summary>
+    public class TestApp : MonoBehaviour
+    {
+        private const string CookieEmoticon = "\uD83C\uDF6A";

```

Comment:
I guess I'm not much fun, but I'd prefer to keep this simple so that we can read it, e.g. "test-cookie".

Reply:
🍪❌